src/helper.h

```diff
@@ -39,6 +45,7 @@ struct get_json_ret_object {
 
 get_json_ret_string get_string(rapidjson::Value const *v, std::string path);
 get_json_ret_int get_int(rapidjson::Value const *v, std::string path);
+get_json_ret_array get_array(rapidjson::Value const &v, std::string path);
 get_json_ret_object get_object(rapidjson::Value const &v, std::string path);
 std::string pretty_print(rapidjson::Document const *v);
```

Comment:
I don't think `pretty_print` is ever used, it should be removed or a comment added to explain why it is here.